.circleci/config.yml

```diff
@@ -58,13 +45,46 @@ jobs:
       - run: npm rebuild node-sass
       - run:
           name: Build the USWDS package
-          command: npm run release
+          command: npm run build
       - run:
           name: Publish to NPM
           command: |
             npm config set "//registry.npmjs.org/:_authToken=$NPM_TOKEN"
             npm publish
 
+  deploy-alpha:
+    <<: *container
+    steps:
+      - checkout
```

Comment:
Note that I'm not restoring the `node_modules` cache here as we're installing in the next step